db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_06_05_085711) do
+ActiveRecord::Schema[6.1].define(version: 2023_06_05_085711) do
```

Comment:
For the narrow purposes of the `7.0` upgrade, I think this is what we want do. I suspect that what will happen is that the next time someone adds a migration post-7.0 we will have some diff noise from bumping this to 7.0 and elsewhere in file with format changes.